client/src/components/footer/footer.js

```diff
@@ -1,9 +1,21 @@
 import React from 'react'
+import AppLogo from "../../images/gamehouse_logo.svg"
+import TeamLogo from "../../images/4baozi_logo.svg"
+import "./styles/footer.css"
 
 export default function Footer() {
     return (
-        <div>
-            This is the footer
+        <div className="footer">
+            <div>
+                <img className="appLogo logoImg" src={AppLogo}/>
```

Comment:
`className` values should have convention of all lowercase with dashes in between